plugins/modules/inventory_intent.py

```diff
@@ -2774,37 +2851,39 @@ def verify_diff_merged(self, config):
         if device_added:
             if not devices_to_add:
                 self.status = "success"
-                msg = "Requested Devices - {0} Added in Cisco DNA Center and Addition verified.".format(str(device_ips))
-                self.log(msg)
+                msg = "Requested Device(s) '{0}' Added in Cisco Catalyst Center and Addition verified.".format(str(device_ips))
+                self.log(msg, "INFO")
             else:
-                self.log("Playbook parameter does not match with Cisco Catalyst Center, meaning device addition task not executed successfully.")
+                self.log("Playbook parameter does not match with Cisco Catalyst Center, meaning device addition task not executed successfully.", "INFO")
```

Comment:
Don't we need to specify which parameter?
self.log("Mismatch between playbook parameter '{0}' and Cisco Catalyst Center '{1}' detected, indicating that the device addition task may not have executed successfully.", "INFO")

Reply:
It's not specific parameter but the playbook input that we are already printing as part of self.want and other failure details are provided as part of merged operation.